app/views/catalog/citation.html.erb

```diff
@@ -0,0 +1,2 @@
+<p>Please verify the citations below with the appropriate style guide.</p>
+<%= render :partial => 'citation' -%>
```

Comment:
Try not to use hash rockets when possible :sweat_smile: 

```suggestion
<%= render partial: 'citation' %>
```

Also there's a small difference between `%>` and `-%>`. The `-%>` will apparently trim the following line break (if the expression leaves a blank line) and there's questions if this actually works (as rails uses erubis to render ERB which might not support this)? Ultimately we tend not care about condensing our html and if we did we would minify it. So typically in Rails world, you hardly ever see the `-%>` or `<%-` usage.

Reply:
You caught me.  Laziness.  Just copied from blacklight.